docs/install-mlrun-ce/index.md

```diff
@@ -0,0 +1,16 @@
+# Installing MLRun CE
+
+
+MLRun Community Edition (CE) provides a complete MLOps platform for managing the full machine learning lifecycle.
```

Comment:
The MLRun Community Edition (CE) provides a complete MLOps platform for managing the full machine learning lifecycle.